content/talent/interview_training.md

```diff
@@ -1,11 +1,275 @@
-## Interview Training
+# Interview Training
 
-[Interview basics] Coming soon
+## Interview basics
 
-[Unconscious bias training] Coming soon
+![Alt Text](https://storage.googleapis.com/sourcegraph-assets/handbook/InterviewBasics/Sourcegraph%20Interview%20Training%20-%20Q2%20FY22.png)
```

Comment:
The text "Alt text" here should be a description of the image for non-sighted users (called alt text, short for alternative text) 🙂